baremaps-cli/src/test/resources/osm-features.csv

```diff
@@ -374,7 +374,7 @@ highway,stop,node,A stop sign
 highway,traffic_signals,node,Lights that control the traffic
 highway,turning_circle,node,"A turning circle is a rounded, widened area usually, but not necessarily, at the end of a road to facilitate easier turning of a vehicle. Also known as a cul de sac."
 highway,toll_gantry,node,"A toll gantry is a gantry suspended over a way, usually a motorway, as part of a system of electronic toll collection. For a toll booth with any kind of barrier or booth see: barrier=toll_booth"
-highway,User Defined,node way,All commonly used values according to Taginfo
+highway,com.baremaps.postgres.User Defined,node way,All commonly used values according to Taginfo
```

Comment:
The changes in this file seem off.

Reply:
Thanks, this is a refactoring issue. This file is not needed anymore so I will simply remove it.